make/modules/java.base/lib/CoreLibraries.gmk

```diff
@@ -209,8 +209,6 @@ endif
 # SapMachine 2023-11-28: build libmallochooks
 
 ifeq ($(call isTargetOs, linux macosx), true)
-  LIBJLI_SRC_DIRS := $(call FindSrcDirsForComponent, java.base, libmallochooks)
```

Comment:
Will you do the same in head?
https://github.com/SAP/SapMachine/blob/84a5dff6bdfbe7ba9885fb9891253869516c788c/make/modules/java.base/lib/CoreLibraries.gmk#L213
Might even be better to fix it there and backport the cleanup. As you prefer.

Reply:
Yes, that will be fixed in head as well.

Reply:
This will be part of https://github.com/SAP/SapMachine/pull/1638, the current PR for jdk-23+17